app/i18n/pt-pt/sub.php

```diff
@@ -0,0 +1,296 @@
+					'help' => 'Example: <code>//script[@type="application/json"]</code>',	// TODO
+				),
+			),
+			'html_xpath' => array(
+				'_' => 'HTML + XPath (Web scraping)',	// IGNORE
+				'feed_title' => array(
+					'_' => 'Título do Feed',
+					'help' => 'Exemplo: <code>//title</code> ou uma string estática: <code>"Meu feed personalizado"</code>',
+				),
+				'help' => '<dfn><a href="https://www.w3.org/TR/xpath-10/" target="_blank">XPath 1.0</a></dfn>é uma linguagem de consulta padrão para usuários avançados e que o FreshRSS suporta para habilitar o Web scraping.',
+				'item' => array(
+					'_' => 'encontrar notícias <strong>items</strong><br /><small>(mais importantes)</small>',
+					'help' => 'Exemplo: <code>//div[@class="news-item"]</code>',
+				),
+				'item_author' => array(
+					'_' => 'Autor do item',
+					'help' => 'Também pode ser uma string estática. Exemplo: <code>"Anônimo"</code>',
+				),
+				'item_categories' => 'Etiquetas do item',
+				'item_content' => array(
+					'_' => 'Conteúdo do item',
+					'help' => 'Exemplo para pegar o item completo: <code>.</code>',
+				),
+				'item_thumbnail' => array(
+					'_' => 'Miniatura do item',
+					'help' => 'Exemplo: <code>descendant::img/@src</code>',
+				),
+				'item_timeFormat' => array(
+					'_' => 'Custom date/time format',	// TODO
+					'help' => 'Optional. A format supported by <a href="https://php.net/datetime.createfromformat" target="_blank"><code>DateTime::createFromFormat()</code></a> such as <code>d-m-Y H:i:s</code>',	// TODO
+				),
+				'item_timestamp' => array(
+					'_' => 'Data do Item',
+					'help' => 'O resultado será parecido com: <a href="https://php.net/strtotime" target="_blank"><code>strtotime()</code></a>',
+				),
+				'item_title' => array(
+					'_' => 'Titulo do Item',
+					'help' => 'Utilize especialmente <a href="https://developer.mozilla.org/docs/Web/XPath/Axes" target="_blank">XPath axis</a> <code>descendant::</code> like <code>descendant::h2</code>',
+				),
+				'item_uid' => array(
+					'_' => 'ID único do item',
+					'help' => 'Opcional. Exemplo: <code>descendant::div/@data-uri</code>',
+				),
+				'item_uri' => array(
+					'_' => 'Link do item (URL)',
+					'help' => 'Exemplo: <code>descendant::a/@href</code>',
+				),
+				'relative' => 'XPath (relativo do item) para:',
+				'xpath' => 'XPath para:',
+			),
+			'json_dotnotation' => array(
+				'_' => 'JSON (notação de ponto)',
+				'feed_title' => array(
+					'_' => 'título do feed',
+					'help' => 'Exemplo: <code>meta.title</code> ou uma string estática: <code>"Meu feed personalizado"</code>',
+				),
+				'help' => 'Um JSON na notação de ponto usa pontos entre os objetos e colchetes para arrays (e.g. <code>data.items[0].title</code>)',
+				'item' => array(
+					'_' => 'encontrando novidades <strong>itens</strong><br /><small>(mais importante)</small>',
+					'help' => 'Caminho do JSON para o array contendo os itens, e.g. <code>$</code> or <code>Novos Items</code>',
```

Comment:
```suggestion
					'help' => 'Caminho do JSON para o array contendo os itens, e.g. <code>$</code> or <code>newsItems</code>',
```

`Novos Items` with a space is not a possible JSON key